gemspec.rb

```diff
@@ -75,27 +76,30 @@ def specification(version, default_adapter, platform = nil)
     lib/gollum-lib/file.rb
     lib/gollum-lib/file_view.rb
     lib/gollum-lib/filter.rb
+    lib/gollum-lib/filter/bibtex.rb
     lib/gollum-lib/filter/code.rb
+    lib/gollum-lib/filter/critic_markup.rb
     lib/gollum-lib/filter/emoji.rb
     lib/gollum-lib/filter/macro.rb
-    lib/gollum-lib/filter/metadata.rb
+    lib/gollum-lib/filter/pandoc_bib.rb
     lib/gollum-lib/filter/plain_text.rb
     lib/gollum-lib/filter/plantuml.rb
     lib/gollum-lib/filter/remote_code.rb
     lib/gollum-lib/filter/render.rb
     lib/gollum-lib/filter/sanitize.rb
     lib/gollum-lib/filter/tags.rb
     lib/gollum-lib/filter/toc.rb
-    lib/gollum-lib/filter/wsd.rb
+    lib/gollum-lib/filter/yaml.rb
     lib/gollum-lib/git_access.rb
-    lib/gollum-lib/gitcode.rb
     lib/gollum-lib/helpers.rb
     lib/gollum-lib/hook.rb
     lib/gollum-lib/macro.rb
     lib/gollum-lib/macro/all_pages.rb
+    lib/gollum-lib/macro/audio.rb
```

Comment:
I don't see `warn.rb`, `note.rb` or `octicon.rb`, does that make sense?

Reply:
The file list in the gemspec will be updated automagically by the `rake build` command (or more precisely, by its `rake gemspec` subtask).